src/CHANGELOG.tsx

```diff
@@ -9,6 +9,7 @@ import { change, date } from 'common/changelog';
 
 // prettier-ignore
 export default [
+  change(date(2021, 2, 27), 'Added German translations in de/messages.json for common, core.abilities, and death.knight sections', maestrohdude),
```

Comment:
The check is failing her because you have not added yourself to the contributors yet. Add yourself in `src/CONTRIBUTORS.ts` with the same format as others and then import your name on line 3 of this file.

Reply:
I'm marking this as resolved because I believe I properly added myself to the contributors file on my patch-1 branch and I've edited the changelog file to now have me imported, but it's not showing the contributors.ts edit in this pull request.